docusaurus/docs-internals/editor-controls/data-controls/editableSelect.md

```diff
@@ -0,0 +1,406 @@
+| `helper?.position` | `"top-start"` \| `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` | `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `choices`          | `Array<Choice>`                                                                                                                                                                            |    -    | `Choice` is an array of objects that are directly provided to the component.<br/><br/>**`Choice: { title: string; value: string; allowRemove?: boolean; allowEdit?: boolean; allowDuplicate?: boolean; }`**<br/><br/> Each `Choice` object defines the following properties:<br/><br/>`title` - Required key, a string representing the display title of the choice.<br/><br/>`value` - Required key, a string value representing the choice. This value is returned when the choice is selected.<br/><br/>`allowRemove?` - Optional key, allows to remove the `Choice`.<br/><br/>`allowDuplicate?` - Optional key, allows to duplicate the `Choice`.<br/><br/>`allowEdit?` - Optional key, allows to edit `title` of the `Choice`. |
+| `onChange`         | `function`                                                                                                                                                                                 |    -    | Changes the value of the choices array, depends on action. <br/><br/> **(action: OnChangeAction)=>void** <br/><br/> Receives the `OnChangeAction` which is <br/><br/> `{type:OnChangeActionType, payload: string}` <br/><br/> where OnChangeActionType in enum <br/><br/> `enum OnChangeActionType {remove="remove", duplicate="duplicate", edit="edit", change="change", reset="reset"`                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
1. replace **(action: OnChangeAction)=>void** with **type Function = (action: OnChangeAction) => void**
2. About: "Changes the value of the choices array, depends on action.". Need to be explained in more details, as example need to specify that this onChange will be callen when choice from select will be selected, when will be duplicate/remove/edit
3. replace `{type:OnChangeActionType, payload: string}` with **type OnChangeAction = { type: "change" | "edit" | "duplicate" | "reset" | "remove" ; payload: string }**